### ISSUE TITLE
Question about openSNP summary data

### ISSUE THREAD

**Issue by sarahcn** [sarahcn/dtc_qc#4]
There were 31 of the 104 SNPs in the JSON file that didn't have a "*relative_values.--" entry. Would those 31 have 0 missing calls?

**Comment by gedankenstuecke** [comment]
Right, if there's no `--` then we didn't have any missed genotyping calls for that SNP.
